### DISCUSSION TITLE
AddQuality wrong results?

### DISCUSSION THREAD

**Discussion by sklein-tu** [marcomusy/vedo#592] [Q&A]
Hello, I was trying to compare the quality of my optimized mesh. But when I use 

`mesh.addQuality(measure=1).cmap('RdYlBu', on='cells').print()`

to get the aspect ratio of my triangles, almost all values are above 1 up to 300. Shouldn't the function produces values between 0 and 1 or am I misunderstanding its purpose?

Thanks in advance

**Comment by marcomusy** [comment]
Hello, it's probably the inverse of what you want. You can add an array with any mathematical operation with eg:
```python
mesh.addQuality(measure=1)
arr = mesh.celldata["Quality"]  # numpy array
mesh.cmap('RdYlBu', 1/arr, on='cells')
```
